apps/src/code-studio/pd/form_components/FormController.jsx

```diff
@@ -363,10 +363,14 @@ export default class FormController extends React.Component {
    * @return {boolean} true if this page is valid, false if any required fields
    *         are missing
    */
-  validateCurrentPageRequiredFields() {
-    const currentPage = this.getCurrentPageComponent();
+  validatePageRequiredFields(pageIndex) {
+    if (pageIndex < 0 || pageIndex >= this.getPageComponents().length) {
+      return;
```

Comment:
Can we `throw` here?  We should never ask to validate a page out of range.

Reply:
Fixed.